exalearn-ui/src/app/components/footer/footer.component.scss

```diff
@@ -2,14 +2,20 @@
 
 .footer {
 	width: 100vw;
-	text-align: center;
 	position: fixed;
 	bottom: 0;
-	height: $footer-height;
+	height: $footer-height + 10px;
```

Comment:
maybe you'll just change $footer-height

Reply:
tried this one first, worked. then changed footer from 30px to 40px. Styles broke, went with this one )

Reply:
fixed